secrets/secrets_test.go

```diff
@@ -63,6 +63,19 @@ func TestGetSecretConfigFilesFormat(t *testing.T) {
 	assert.Equal(t, true, run.WorkerConfig.MustGet("from-secret"), "value for from-secret")
 }
 
+func TestGetSecretConfigFilesFormat(t *testing.T) {
+	runnercfg, run := setup(t)
+
+	tc.FakeSecretsCreateSecret("worker-pool:pp/wt", &tcsecrets.Secret{
+		Secret: []byte(`{"config": {"from-secret": true}}`),
```

Comment:
Should we put `files` in here or wait?

Reply:
That's a stlil-to-be-implemented feature, so "wait".

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1561957